ui/src/components/facets/FacetHistogram.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from "react";
```

Comment:
Please change this file to indent with 2 spaces instead of 4. That's what our other js files do.